src/style.css

```diff
@@ -1,3 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-inline: 50px;
+  align-items: center;
+  position: relative;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  align-items: center;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
```

Comment:
It's not recommended to style HTML elements directly (except for `html` and `body`). Instead, you should add a class to the `a` element inside `.nav__link` and style that class. (CSS checklist)